content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,62 @@ menu:
 weight: 101
 ---
 
+## v2.0.9 [2021-09-23]
+
+### Features
+
+- **Flux updates**:
+  - Update to [Flux v0.130.0](/flux/v0.x/release-notes/#v01300-2021-09-15).
+  - Add support for [`influxdb.cardinality()`](/flux/v0.x/stdlib/influxdata/influxdb/cardinality/) function.
+
+- **Improved logging**:
+  - Add additional logging to Flux end to end tests (`TestFluxEndToEnd`) to help diagnose test failures.
+  - Add `--flux-log-enabled` option to [`influxd`](/influxdb/v2.0/reference/config-options/) to show detailed logs for Flux queries.
+
+- **Performance optimizations**:
+  - Optimize series iteration for queries that can be answered without inspecting TSM data.
+  - Optimize queries with predicates that contain multiple measurements.
+
+- **API update**: Add a new route `/api/v2/resources` that returns a list of known resources to the platform, including the following resource types:
+	- `AuthorizationsResourceType`
+	- `BucketsResourceType`
+	- `DashboardsResourceType`
+	- `OrgsResourceType`
+	- `SourcesResourceType`
+	- `TasksResourceType`
+	- `TelegrafsResourceType`
+	- `UsersResourceType`
+	- `VariablesResourceType`
+	- `ScraperResourceType`
+  - `SecretsResourceType`
+	- `LabelsResourceType`
+	- `ViewsResourceType`
+	- `DocumentsResourceType`
+	- `NotificationRuleResourceType`
+	- `NotificationEndpointResourceType`
+	- `ChecksResourceType`
+	- `DBRPResourceType`
+
```

Comment:
@glinton @lesam @russorat, do we want to say anything more about why/how this allows the ui (and cli?) to generate an all-access token in a more uniform, and less brittle, way between cloud and oss?

Reply:
it might be a good place to point out that an all-access token, in it's current form, is a snapshot of the resources at that point in time and is not evolutionary. meaning that if a resource type (generic bucket permission covers new instances of the same type) is added after creation, that "all-access" token will not include permissions to interact with that new resource.